data/users/__init__.py

```diff
@@ -69,6 +69,9 @@ def get_users_handler(config, _, override_config_dir, oauth_login):
         network_timeout = config.get("LDAP_NETWORK_TIMEOUT")
         ldap_user_filter = config.get("LDAP_USER_FILTER", None)
         ldap_superuser_filter = config.get("LDAP_SUPERUSER_FILTER", None)
+        ldap_global_readonly_superuser_filter = config.get(
+            "LDAP_GLOBAL_READONLY_SUPERUSER_FILTER", None
```

Comment:
Which one will take precedence if both `LDAP_SUPERUSER_FILTER` and `LDAP_GLOBAL_READONLY_SUPERUSER_FILTER` are defined?

Reply:
So that means that the same user will always have the superset of permissions? In the example above, if defined in both group, the user would still have write permissions, not just readonly, correct?

Reply:
It would be a superset, that same user would have superuser permissions and also global readonly permissions